src/main/java/edu/cornell/kfs/sys/service/impl/DisposableClientServiceImplBase.java

```diff
@@ -27,13 +28,20 @@ protected Client getClient() {
                 if (jerseyClient == null) {
                     ClientConfig clientConfig = new ClientConfig();
                     jerseyClient = ClientBuilder.newClient(clientConfig);
+                    if (classToRegister != null) {
+                        jerseyClient.register(classToRegister);
```

Comment:
The ClientConfig object also has a register() method, if you want to register this class in advance at the ClientConfig level.